staging/src/kubevirt.io/api/core/v1/defaults.go

```diff
@@ -195,7 +195,7 @@ func DefaultSlirpNetworkInterface() *Interface {
 	iface := &Interface{
 		Name: "default",
 		InterfaceBindingMethod: InterfaceBindingMethod{
-			Slirp: &InterfaceSlirp{},
+			DeprecatedSlirp: &DeprecatedInterfaceSlirp{},
```

Comment:
Should the `DefaultSlirpNetworkInterface` be removed?

Reply:
Yes, it should be removed as well. Thanks.

Reply:
Done